ci/builders/README.md

```diff
@@ -154,231 +143,237 @@ An archive component is used to tell the recipes which artifacts are generated b
-
-```
-          "tests": [
-            {
-                "language": "python3",
-                "name": "Host Tests for host_debug_impeller_vulkan",
-                "parameters": [
-                    "--variant",
-                    "host_debug_impeller_vulkan",
-                    "--type",
-                    "impeller-vulkan",
-                    "--engine-capture-core-dump"
-                ],
-                "script": "flutter/testing/run_tests.py",
-                "type": "local"
-            }
-        ]
+In the example above the ninja command will use the configuration for
+host\_debug and will build artifacts and embedder targets as described
+by the
+[flutter/build/archives/BUILD.gn](https://cs.opensource.google/flutter/engine/+/master:build/archives/BUILD.gn)
+file.
+
+#### Tests
+
+This section of the build configuration is also known as local tests. It
+contains a list of dictionaries with configurations for scripts and
+parameters used to run tests inside the current build unit. These tests
+should not reference or use anything outside of the commit checkout or
+the outputs generated by running gn and ninja sections of the build
+configuration.
+
+```yaml
+"tests": [
+   {
+       "language": "python3",
+       "name": "Host Tests for host_debug_impeller_vulkan",
+       "parameters": [
+           "--variant",
+           "host_debug_impeller_vulkan",
+           "--type",
+           "impeller-vulkan",
+           "--engine-capture-core-dump"
+       ],
+       "script": "flutter/testing/run_tests.py"
+   }
+]
 ```
 
 Description of the fields:
 
-* **Language,** the executable used to run the script, e.g. python3.
-* **Name,** the name of the step running the script.
-* **Parameters**, flags or parameters passed to the script.
-* **Script**, the path to the script to execute relative to the checkout directory.
-* **Type**, the test type. (Deprecate? Test on the build config will be always local)
+* **language** - the executable used to run the script, e.g. python3, bash.
+* **name** - the name of the step running the script.
+* **parameters** - flags or parameters passed to the script.
+* **Script** - the path to the script to execute relative to the checkout
+directory.
 
-The test scripts will run in a deferred context (failing the step only after logs have been uploaded). The tester and builder recipes provide an environment variable called FLUTTER\_LOGS\_DIR pointing a temporary directory where the test runner can place any logs|artifacts needed to debug issues. At the end of the test execution the content of FLUTTER\_LOGS\_DIR will be uploaded to Google Cloud Storage before signaling the pass | fail test state.
+The test scripts will run in a deferred context (failing the step only after
+logs have been uploaded). Tester and builder recipes provide an environment
```

Comment:
Do stdout and stderr need to be manually captured in this way, or are they captured automatically?

Reply:
Stdout and stderror are captured and presented as part of the step execution. Only additional logs need to be captured and saved separately, e.g. screenshots, videos, device dumpsys or host state.

Reply:
Might be good to note here to avoid confusion.